### PR TITLE
feat(council): Add "Council" Module with Associated Initial Doctypes  

### PR DESCRIPTION
**Pull request by Ali-Alnosairi**
### **Description:**
This pull request adds the initial set of doctypes for the "Council" module in the Academia application. These doctypes lay the foundation for managing council-related activities, including council details, council members, sessions, topics, assignments, and templates. While the module is still a work in progress, this pull request serves as an initial step towards implementing the Council module.

### **Changes Made:**

Created a new module named "Council" within the Academia application.
Added the following doctypes:
"Council": Represents the council details and settings.
"Council Member": Stores information about council members.
"Session": Manages council sessions and their associated details.
"Session Topic Assignment": Assigns topics to sessions.
"Session Member": Tracks council members participating in sessions.
"Topic": Manages topics within the council context.
"Topic Applicant": Stores information about individuals applying for topics.
"Topic Assignment": Assigns topics to council members.
"Topic Assignment Attachment": Allows attachment of files or documents to topic assignments.
"Topic Template": Defines predefined templates for topics.

### **Additional Information:**

This pull request marks the initial implementation of the Council module in the Academia application.
The doctypes have been created with basic fields and initial validations, but further enhancements and functionality are yet to be added.
Future work includes implementing business logic, permissions, forms, views, and integrating the Council module with other parts of the Academia application.

### Reviewers: @BakrAldubai

### CODE REVIEW

**Comment by BakrAldubai** [comment]
Thank you so much @Ali-Alnosairi .  We appreciate your contributions. we will review your comment and replay to you as soon as possible.

File: academia/modules.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-Academia
\ No newline at end of file
+Academia
+Council
```
Comment (BakrAldubai) [inline academia/modules.txt:2]: Rename the module into "Councils"

File: academia/council/doctype/topic_template/topic_template.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ "field_order": [
+  "name1",
+  "description"
+ ],
+ "fields": [
+  {
+   "fieldname": "name1",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Name",
```
Comment (BakrAldubai) [inline academia/council/doctype/topic_template/topic_template.json:19]: delete the name field and make the DocType naming by Prompt

**Review by BakrAldubai — CHANGES_REQUESTED** [review]

File: academia/modules.txt (hunk 1 above)
Comment (Ali-Alnosairi) [reply academia/modules.txt:2]: Thank you for your feedback. I will rename the module from 'Council' to 'Councils' as requested. I'll make the necessary changes and update the pull request accordingly. Please re-review it once the updates are complete. Thank you for your time.

File: academia/council/doctype/topic_template/topic_template.json (hunk 2 above)
Comment (Ali-Alnosairi) [reply academia/council/doctype/topic_template/topic_template.json:19]: Thank you for your feedback. I will delete the 'name' field and implement a prompt-based mechanism for dynamic DocType naming as requested. I'll make the necessary changes and update the pull request accordingly. Please re-review it once the updates are complete. Thank you for your patience.

File: academia/modules.txt (hunk 1 above)
Comment (Ali-Alnosairi) [reply academia/modules.txt]: changes  applied

**Review by BakrAldubai — COMMENTED** [review]
Good Job

**Pull request merged**